build.assets/tooling/go.mod

```diff
@@ -69,6 +69,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
+require github.com/spf13/afero v1.9.2
```

Comment:
Move this up into the main require block. (The `go` tool doesn't do this for you sometimes, but you can just move the line manually)